.github/workflows/test.yml

```diff
@@ -0,0 +1,25 @@
+name: Run all tests & checks
```

Comment:
I think we probably don't need this, as we're running dbt test here:

https://github.com/openedx/aspects-dbt/blob/main/.github/workflows/coverage.yml#L47

Since dbt needs ClickHouse running, and alembic to have been run, we're just doing these tests in the Tutor / Aspects context there.